clog/loggers.py

```diff
@@ -269,7 +269,7 @@ def close(self):
             self.stream_files[name].close()
 
     def _create_file(self, stream):
-        return open(os.path.join(config.log_dir, stream + '.log'), 'ab', 0)
+        return open(os.path.join(config.log_dir.value, stream + '.log'), 'ab', 0)
```

Comment:
more generally, these should be `text_type(config_dir)` since they aren't necessarily `ValueProxy` objects.

But maybe this should just get fixed in staticconf itself (staticconf should implement the `__path__` or whatever the function is)